### DISCUSSION TITLE
Will Filament Forms overwrite files that already came with Filament Admin?!

### DISCUSSION THREAD

**Discussion by C4NH4M** [filamentphp/filament#7496] [General]
Hi, Firstly, Filament is a god send. :)

I have installed the 2.x package of the admin panel with all the other packages baked in and wish to stick with it till V3 has been around a while.

My question is, If I now install the Filament forms with it affect my existing files that were created with the Filament Admin install?

From Filament:
#These commands will ruthlessly overwrite existing files in your application, hence why we only recommend using this method for new projects.

Sorry if it seems an obvious question but am new to Filamentphp.

:)

**Comment by danharrin** [comment]
Installing the forms package will only overwrite your frontend code, shouldn't touch anything in the admin panel. However, you should be using version control so even if something goes wrong, you can easily revert it.

**Reply by C4NH4M** [reply]
@danharrin - Many Thanks. Your products are amazing. Have a good day :)
